packages/test/test-end-to-end-tests/package.json

```diff
@@ -78,6 +78,7 @@
 		"temp-directory": "nyc/.nyc_output"
 	},
 	"dependencies": {
+		"@ff-internal/aria-logger": "^0.0.20",
```

Comment:
Added aria logger because of a suspicion that it might be related to the timeout. No hard evidence so far. Note that I didn't want to actually send telemetry to Aria with it, so after installing it I went to node_modules/@ff-internal/aria-logger/dist/ariaLogger.js and modified it so it didn't actually send anything to a remote endpoint.